### ISSUE TITLE
Is your rotated images also being used to compute real/fake logits?

### ISSUE THREAD

**Issue by XavierXiao** [vandit15/Self-Supervised-Gans-Pytorch#1]
In the paper, only upright images are used to compute real/fake logits, but it seems like in your code, images are packed with their rotations to feed into the discriminator, and I wonder where the rotated images are excluded from computing the real/fake logits?
